mlrun/feature_store/api.py

```diff
@@ -121,31 +123,47 @@ def get_offline_features(
-    :param target:         where to write the results to
-    :param drop_columns:   list of columns to drop from the final result
+    :param feature_vector:          feature vector uri or FeatureVector object. passing feature vector obj requires
+                                    update permissions
+    :param entity_rows:             dataframe with entity rows to join with
+    :param target:                  where to write the results to
+    :param drop_columns:            list of columns to drop from the final result
     :param entity_timestamp_column: timestamp column name in the entity rows dataframe
-    :param run_config:     function and/or run configuration
-                           see :py:class:`~mlrun.feature_store.RunConfig`
-    :param start_time:      datetime, low limit of time needed to be filtered. Optional.
-        entity_timestamp_column must be passed when using time filtering.
-    :param end_time:        datetime, high limit of time needed to be filtered. Optional.
-        entity_timestamp_column must be passed when using time filtering.
-    :param with_indexes:    return vector with index columns and timestamp_key from the feature sets (default False)
-    :param update_stats:    update features statistics from the requested feature sets on the vector. Default is False.
-    :param engine:          processing engine kind ("local", "dask", or "spark")
-    :param engine_args:     kwargs for the processing engine
-    :param query:          The query string used to filter rows
+    :param run_config:              function and/or run configuration
+                                    see :py:class:`~mlrun.feature_store.RunConfig`
+    :param start_time:              datetime, low limit of time needed to be filtered. Optional.
+                                    entity_timestamp_column must be passed when using time filtering.
+    :param end_time:                datetime, high limit of time needed to be filtered. Optional.
+                                    entity_timestamp_column must be passed when using time filtering.
+    :param with_indexes:            return vector with index columns and timestamp_key from the feature sets
+                                    (default False)
+    :param update_stats:            update features statistics from the requested feature sets on the vector.
+                                    Default is False.
+    :param engine:                  processing engine kind ("local", "dask", or "spark")
+    :param engine_args:             kwargs for the processing engine
+    :param query:                   The query string used to filter rows
+    :param relations:                A Dictionary that indicate all the relations between different feature sets.
+                                    The format is
+                                    {"feature_set_name_1:feature_set_name_2":{"column_of_1":"column_of_2",...}...}
```

Comment:
I don't understand why would the value of the main dict should be a dictionary instead of a simple list so it will be `Dict[str, List[str]]` where the keys are the feature sets involved and the values are list of the columns to take like:

```
{
    "feature_set_1 : feature_set_2": ["column_of_1 : column_of_2", ...],
    ...
}
```

It is more consistent with how the feature sets names are being passed, no?

Side note: I would also allow spaces between each feature set / column so it will be more readable. When parsing it I'll remove all spaces and then split by `':'`